app/views/appointment_summaries/email_confirmation.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="page-header">
+  <h1>Make sure this is right</h1>
+</div>
+
+
+<%= form_for @appointment_summary,
+             url: preview_appointment_summaries_path,
+             method: :post do |f| %>
+  <%= error_messages_for(@appointment_summary) %>
+
+  <% (AppointmentSummary.columns.map(&:name) - %w(id created_at updated_at user_id email)).each do |field| %>
```

Comment:
It would help if this was behind something intention-revealing in `AppointmentSummary`.